### PR TITLE
Rename github clone paths from 'git@github.com:' to 'https://github.c…

### PR DESCRIPTION
**Pull request by lpopov**
…om/'.

Remove '--provider virtualbox' from 'vagrant init' command in 2.4.0.0.
Remove cp/mv commands in 3.2.0.0 (.vagrant directory stays the same when changing branches).
Some small fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ You can download the manual from [Leanpub](http://leanpub.com/pragmatic_virtuali
 
 ## Get the extra content
 
-    git clone git@github.com:damko/pragmatic_virtualization_extra.git
+    git clone https://github.com/damko/pragmatic_virtualization_extra.git
```
Comment (damko) [inline README.md:10]: sounds good

File: manuscript/ch1/1.1.0.0-get_ready.txt [hunk 2]
```diff
@@ -25,7 +25,7 @@ Once Git is installed you are ready to download the _extra_ repository:
 {linenos=off, lang=bash}
 ~~~~~~~
     cd <your-favorite-directory>
-    git clone git@github.com:damko/pragmatic_virtualization_extra.git
+    git clone https://github.com/damko/pragmatic_virtualization_extra.git
 ~~~~~~~
```
Comment (damko) [inline manuscript/ch1/1.1.0.0-get_ready.txt:30]: sounds good

File: manuscript/ch2/2.4.0.0-create_vagrant_box_from_cloud.txt [hunk 3]
```diff
@@ -17,7 +17,7 @@ To use this box run:
 {linenos=off, lang=bash}
 ~~~~~~~
 cd $pve/ariadne/orchestration/vagrant
-vagrant init deb/jessie-amd64 --provider virtualbox
+vagrant init deb/jessie-amd64
 ~~~~~~~
```
Comment (damko) [inline manuscript/ch2/2.4.0.0-create_vagrant_box_from_cloud.txt:21]: this is correct :+1:

File: manuscript/ch3/3.2.0.0-improving_vagrantfile.txt [hunk 4]
```diff
@@ -6,12 +6,7 @@ The Vagrantfile can be written to configure several virtual machines and to host
 
 The _improving_vagrantfile_ _extra_ branch has an improved version of the previous Vagrantfile but before doing anything you need to backup your `.vagrant` directory because it contains some important and unique information about your vagrant virtual machine.
 
-{linenos=off, lang=bash}
-~~~~~~~
-cp -fR $pve/ariadne/orchestration/vagrant/.vagrant /tmp/
-~~~~~~~
-
-Now you should checkout the _create_vagrant_box_from_iso_ branch from the _extra_ repository:
+You should checkout the _improving_vagrantfile_ branch from the _extra_ repository:
```
Comment (damko) [inline manuscript/ch3/3.2.0.0-improving_vagrantfile.txt:9]: Ah!! I couldn't remember why I added the .vagrant backup but now I get it. There was a time in which I forgot to add .vagrant to .gitignore so switching the branch would overwrite the local .vagrant directory. But this issue has been fixed long time ago. :-) thanks for noticing this

File: manuscript/ch2/2.4.0.0-create_vagrant_box_from_cloud.txt (hunk 3 above)
Comment (lpopov) [inline manuscript/ch2/2.4.0.0-create_vagrant_box_from_cloud.txt:21]: With the "--provider" option, it gives me an error (Vagrant 1.7.2) -  "An invalid option was specified. The help for this command is available below.". And VirtualBox is the default provider, so no need to set it explicitly - https://docs.vagrantup.com/v2/providers/default.html

Comment (damko) [inline manuscript/ch2/2.4.0.0-create_vagrant_box_from_cloud.txt:21]: I believe that but I've also some fuzzy memories about that option. I'm just going through the whole packer build process to double check before merging :-)

**Pull request closed without merging**